### PR TITLE
fix(map-chat): disable dummy info window

### PR DESCRIPTION
**Pull request by geminiyellow**
## Short Description:

`msg.text = html_sanitize(msg.text);` return `undefined` as `'undefined'`.

## Fixes:

disable `just appeared` info window

### CODE REVIEW

File: map-chat/javascript/map.js [hunk 1]
```diff
@@ -320,9 +320,8 @@ function displayMessageOnMap(msg, olat, olong, sessid, icostr, expir, pokenick)
       }
     });
 
-    if (msg.text) {
+    if (msg.text !== 'undefined') {
```
Comment (solderzzc) [inline map-chat/javascript/map.js]: display the location is not that bad...

Comment (geminiyellow) [inline map-chat/javascript/map.js]: @solderzzc yep, display the location is not bad, but the infoWindow is meaningless, i dont think human can read it. you can try it by yourself. btw, the same info is output in console.debug, too. 
and, if you want a infoWindow, i think there should have a filter.just pop up what you want.

**Comment by solderzzc** [comment]
I've understood what you meant, thanks a lot. 
👍

**Comment by solderzzc** [comment]
I'd like to have the dummy info back, till we find a better GUI. 
Since the box shows how fast the pokemon appear. 
Or make a timeout to clean up?

**Pull request merged**